frontend/src/pages/api/hello.ts

```diff
@@ -1,13 +1,18 @@
 // Next.js API route support: https://nextjs.org/docs/api-routes/introduction
-import type { NextApiRequest, NextApiResponse } from 'next'
+import type { NextApiRequest, NextApiResponse } from "next";
+
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
 
 type Data = {
-  name: string
-}
+	name: string;
+};
 
 export default function handler(
-  req: NextApiRequest,
-  res: NextApiResponse<Data>
+	req: NextApiRequest,
+	res: NextApiResponse<Data>
 ) {
-  res.status(200).json({ name: 'John Doe' })
+	const users = prisma.user.findMany();
+	console.log(JSON.stringify(users));
+	res.status(200).json({ name: "John Doe" });
```

Comment:
I have not tried this. But I assume this should be removed. Prisma should not be called in the frontend